docgen/middlewares.js

```diff
@@ -1,22 +1,25 @@
-import layouts from 'metalsmith-layouts';
 import headings from 'metalsmith-headings';
-import navigation from 'metalsmith-navigation';
+import layouts from 'metalsmith-layouts';
 import markdown from 'metalsmith-markdown';
+import msWebpack from 'ms-webpack';
+import navigation from 'metalsmith-navigation';
```

Comment:
Woah we kept that module, I thought it was really hard to work with it. But if it works now, wfm